plugins/entando-plugin-jacms/src/main/java/org/entando/entando/plugins/jacms/web/contentmodel/ContentTypeResource.java

```diff
@@ -0,0 +1,165 @@
+            @ApiResponse(code = 201, message = "Created"),
+            @ApiResponse(code = 401, message = "Unauthorized"),
+            @ApiResponse(code = 403, message = "Forbidden"),
+            @ApiResponse(code = 404, message = "Not Found") })
+    @PutMapping("/plugins/cms/contentTypes")
+    ResponseEntity<ContentTypeDto> update(
+            @ApiParam(value = "contentType", required = true) @Valid @RequestBody ContentTypeDtoRequest contentType,
+            BindingResult bindingResult);
+
+
+    @RestAccessControl(permission = Permission.SUPERUSER)
+    @GetMapping("/plugins/cms/contentTypeAttributes")
+    ResponseEntity<PagedRestResponse<String>> getContentTypeAttributeTypes(RestListRequest requestList);
+
+
+    @RestAccessControl(permission = Permission.SUPERUSER)
+    @GetMapping("/plugins/cms/contentTypeAttributes/{attributeTypeCode}")
+    ResponseEntity<SimpleRestResponse<AttributeTypeDto>> getContentTypeAttributeType(
+            @PathVariable String attributeTypeCode);
+
+
+    @RestAccessControl(permission = Permission.SUPERUSER)
+    @GetMapping("/plugins/cms/contentTypes/{contentTypeCode}/attributes/{attributeCode}")
+    ResponseEntity<RestResponse<EntityTypeAttributeFullDto, Map>> getContentTypeAttribute(
+            @PathVariable String contentTypeCode,
+            @PathVariable String attributeCode);
+
+
+    @RestAccessControl(permission = Permission.SUPERUSER)
+    @PostMapping("/plugins/cms/contentTypes/{contentTypeCode}/attributes")
+    ResponseEntity<RestResponse<EntityTypeAttributeFullDto, Map>> addContentTypeAttribute(
+            @PathVariable String contentTypeCode,
+            @Valid @RequestBody EntityTypeAttributeFullDto bodyRequest,
+            BindingResult bindingResult);
+
+
+    @RestAccessControl(permission = Permission.SUPERUSER)
+    @PutMapping("/plugins/cms/contentTypes/{contentTypeCode}/attributes/{attributeCode}")
+    ResponseEntity<RestResponse<EntityTypeAttributeFullDto, Map>> updateContentTypeAttribute(
+            @PathVariable String contentTypeCode,
+            @PathVariable String attributeCode,
+            @Valid @RequestBody EntityTypeAttributeFullDto bodyRequest,
+            BindingResult bindingResult);
+
+
+    @RestAccessControl(permission = Permission.SUPERUSER)
+    @DeleteMapping("/plugins/cms/contentTypes/{contentTypeCode}/attributes/{attributeCode}")
+    ResponseEntity<SimpleRestResponse<Map>> deleteContentTypeAttribute(
+            @PathVariable String contentTypeCode,
+            @PathVariable String attributeCode);
+
+
+    @RestAccessControl(permission = Permission.SUPERUSER)
+    @PostMapping("/plugins/cms/contentTypes/refresh/{contentTypeCode}")
+    ResponseEntity<SimpleRestResponse<Map>> reloadReferences(@PathVariable String contentTypeCode);
+
+//    @RestAccessControl(permission = Permission.SUPERUSER)
+//    @PostMapping("/plugins/cms/contentTypesStatus")
+//    ResponseEntity<SimpleRestResponse<Map>> reloadReferences(
```

Comment:
Remove commented code if it is not used.

Reply:
This one was in progress, it's implemented now